apps/site-landing/src/containers/HelpDocs/FaqList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { FaqItem } from "./FaqItem";
+import { Header } from "semantic-ui-react";
+import { FAQDocument } from "components/Prismic/types";
+import { useSelector } from "react-redux";
+import { selectLocale } from '@thecointech/shared/containers/LanguageProvider/selector';
+import styles from "./styles.module.less";
+import { FaqMenu } from "./FaqMenu";
+import { Dictionary } from "lodash";
+import { Decoration } from "components/Decoration";
+
+type Props = {
+  title?: string,
+  faqs: FAQDocument[],
+  menu: Dictionary<FAQDocument[]>
+}
+
+export const FaqList = ({ title, faqs, menu }: Props) => {
+  const { locale } = useSelector(selectLocale);
+  return (
+    <>
+    <div className={styles.containerFaq}>
+      <FaqMenu categories={menu} />
+      <Header as="h2" className={"x10spaceBefore"}>
+        <Header.Content>
+          {title ? ((title.split("-"))[1]) : ""}
+        </Header.Content>
+      </Header>
+      {faqs.filter(faq => locale === ((faq.lang!).split("-"))[0]).map(faq => (<FaqItem key={faq.id} {...faq} />))}
```

Comment:
When might this `lang` assertion here be wrong?  Could we just change the type of it to `string` (so we don't need the `!`)

Also, a general style note, it's a lot easier to have only one root-level scope on a line.  If you split `filter` and `map` onto different lines, it would be *way* easier to see what is happening, and also to know what applies to what

Reply:
```suggestion
      {
        faqs
          .filter(faq => locale === faq.lang?.split("-")[0])
          .map(faq => <FaqItem key={faq.id} {...faq} />)
      }
```

Reply:
again, no idea if that even compiles.  But as far as I can tell, it does the same thing